lms/templates/discovery/result_item.underscore

```diff
@@ -0,0 +1,25 @@
+<article class="course" role="region" aria-label="<%= content.display_name %>">
+    <a href="/courses/<%- course %>/info">
+        <header class="course-image">
+            <div class="cover-image">
+                <img src="<%- image_url %>" alt="<%= content.display_name %> <%= content.number %>" />
+                <div class="learn-more" aria-hidden=true><%= gettext("LEARN MORE") %></div>
+            </div>
+        </header>
+        <section class="course-info" aria-hidden=true>
+            <h2 class="course-name">
```

Comment:
Since this ```<h2>``` is in an aria-hidden region, it will not show up in the list of Headings that screen reader users use to navigate the page.  I guess its OK since you have the aria-label on the article element above.  They can also navigate by list of links.  Just something to think about.  It's always very tempting to wrap large blocks of text in hyperlinks, but it can be difficult to make it usable for screen readers. I think we're still fine here.